modules/network-observability-viewing-alerts.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-alert-dashboard_{context}"]
+= Viewing the alert dashboard
+
+You can access alerts for the Network Observability Operator dashboard in the *Alerting* page in the web console.
+
+.Prerequisites
+
+* You have the Network Observability Operator installed.
+//* You have access to the cluster as a user with the `cluster-admin` role or with view permissions for all projects. --question-is this feature for admins only?
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Observe* → *Alerting*.
+
+. From the *Alerts* list, select the *NetObservLokiError* alert.
+//should the name of this alert be NetworkObservabilityLokiHealthError or just LokiHealthError? AFAIK we don't usually use NetObserv downstream.  
+
+. Information about the alert is displayed on the page. Alerts are generated in the folling cases:
+* If the `flowlogs-pipeline` is dropping flows because of Loki errors
+* If the Loki ingestion rate limit has been reached
```

Comment:
The two points there are the same situation, we base our loki erros detection to flows getting dropped after several unsuccessful attempt to push flows to loki.

The second alert case happens when no flows at all are getting ingested for a certain amount of time.